From 58e70152a1bdaf2cc634e547f5400e22708cb316 Mon Sep 17 00:00:00 2001
From: Jacob Chen <jacob2.chen@rock-chips.com>
Date: Tue, 13 Sep 2016 13:52:57 +0800
Subject: [PATCH] dt-bindings: soc: rockchip: add ums mode

On upstream uboot, we use ums mode to update firmware.
Add this flag to help enter USB Mass Storage mode.

Change-Id: I0e515bfd8703bd48d950b72787b365226af11ce9
Signed-off-by: Jacob Chen <jacob2.chen@rock-chips.com>
---
 include/dt-bindings/soc/rockchip_boot-mode.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/dt-bindings/soc/rockchip_boot-mode.h b/include/dt-bindings/soc/rockchip_boot-mode.h
index cb7a60ab4b89..816a83b085ad 100644
--- a/include/dt-bindings/soc/rockchip_boot-mode.h
+++ b/include/dt-bindings/soc/rockchip_boot-mode.h
@@ -13,5 +13,7 @@
 #define BOOT_FASTBOOT		(REBOOT_FLAG + 9)
 /* enter charging mode */
 #define BOOT_CHARGING		(REBOOT_FLAG + 11)
+/* enter usb mass storage mode */
+#define BOOT_UMS		(REBOOT_FLAG + 12)
 
 #endif
-- 
2.35.3

